scripts/zkit_test_update_state.sh

```diff
@@ -0,0 +1,38 @@
+set -e
+# EigenZKit
+
+CIRCUIT=update_state_verifier
+CIRCUIT_DIR=$(cd $(dirname $0);cd ../circuits;pwd)
+#CUR_DIR=$(cd $(dirname $0);pwd)
+ZKIT="/home/ubuntu/workspace/ieigen/EigenZKit/target/release/zkit"
```

Comment:
ZKIT=`which zkit` # Add an readme and let the user to install the zkit manually, and set the zkit in $PATH